.github/workflows/ci.yml

```
@@ -75,6 +76,18 @@ jobs:
           version: ${{ env.GOLANGCI_VERSION }}
```

To avoid duplicating golangci-lint config:
```suggestion
          version: ${{ env.GOLANGCI_VERSION }}
          args: --config=${{ github.workspace }}/.golangci.yaml
```

---

fixed